src/Command/Project/ProjectCreateCommand.php

```diff
@@ -78,13 +96,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
+        // Grab the url of the yaml file.
+        if (!empty($template_options['catalog_url'])) {
+            $options['catalog'] = $template_options['catalog_url'];
+        }
+        else if (!empty($template_options['catalog_url'])) {
+            $options['catalog'] = $template_options['template_url'];
+        }
```

Comment:
the block of above 3 lines (103-105) is a copy of the previous condition (lines 100-102)

Reply:
It's not a straight copy paste, one sets catalog_url and one sets template_url.